apps/projection/src/app/ui/list-item/list-item.component.ts

```diff
@@ -1,35 +1,21 @@
-import { Component, Input } from '@angular/core';
-import { StudentStore } from '../../data-access/student.store';
-import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
-
+import { Component, EventEmitter, Input, Output } from '@angular/core';
 @Component({
   selector: 'app-list-item',
   template: `
     <div class="border border-grey-300 py-1 px-2 flex justify-between">
       {{ name }}
```

Comment:
to make it more shareable, leave the information you want to put inside here to the parent

Reply:
ok, so put an ng-content in the list item where {{name}} is and then put the name in the specific card component?